tests/test_usb_detection.py

```
@@ -0,0 +1,104 @@
+    print("\n" + "=" * 60)
+    print("🔍 Monitoring for USB events... (Press Ctrl+C to stop)")
+    print("=" * 60)
+    
+    monitor = USBMonitor(on_device_event)
+    
+    if not monitor.start():
+        print("\n❌ Failed to start USB monitor!")
+        sys.exit(1)
+    
+    # Handle Ctrl+C gracefully
+    def signal_handler(sig, frame):
```

Ruff (0.14.10)

Unused function argument: `sig`

(ARG001)